src/AltinnCore/Common/Services/Implementation/RepositorySI.cs

```diff
@@ -170,14 +170,15 @@ public bool CreateServiceMetadata(ServiceMetadata serviceMetadata)
         /// <summary>
         /// Creates the application metadata file
         /// </summary>
-        /// <param name="org">the application owner</param>
-        /// <param name="appName">the application name</param>
-        public void CreateApplication(string org, string appName)
+        /// <param name="org">The organisation code for the application owner</param>
+        /// <param name="app">The application name, e.g. "app-name-with-spaces"</param>
+        /// <param name="appTitleNb">The application title, e.g. "app name with spaces", at the moment it get stored as Norwegian title ("nb")</param>
```

Comment:
ingen grunn til å legge på nb. ville heller lagt til at input vil blu brukt som default languge

Reply:
David suggested to use appTitleNb so we are aware it get stored as "nb" appTitle since the code actually only do this. When this has been fixed to be stored as other languages on creation, we will change this variable name. The "nb" parameter had many other dependencies in codebase.

Reply:
Fixed with appTitle, and comment on "nb" default title language.